src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -210,6 +308,11 @@ def report_performance(
     # Set index to include branch configuration/scenario columns
     perf_df, scenario_cols = set_index_scenario_cols(perf_df)
 
+    try:
+        append_perf_data_to_central_logs(perf_df, output_directory)
```

Comment:
What is the intended behavior with respect to running `vipin` as a standalone CLI? 

It looks to me like if you run `vipin` manually, it will still try to append the results to the central logging, unless I'm missing something. In that case, I think that the output directory may not necessarily conform to the directory structure you're relying on above to fill runner information.

Reply:
(IMO I think it would make sense to not do the central logging for manual `vipins`, again assuming you haven't already excluded them somehow)

Reply:
If you want to keep the CLI `vipin` from logging to the central directory, you could probably move the call `append_perf_data_to_central_logs` to the inside of `try_run_vipin` in `runner.py` (ln 108ish)

You could make `report_performance` return the performance df, then pass it into your new function that way, instead of calling it inside report_performance. Then psimulate would use it, but `vipin` wouldn't. 

Alternatively, if you want to let vipin be able to do this logging you could make it a CLI argument, and use the input directory instead of the output_directory--since `psimulate` is using the same `log_directory` for both of these, you should be able to just use that instead in `append_perf_data_to_central_logs` 

Would either of those work?

Reply:
I like the first idea. I think it could also give us more direct access to some of the info we need in the runner file. PR updates pending

Reply:
I've moved the append logic to psimulate/runner.py the way you suggested.